otx/algorithms/common/adapters/mmcv/models/backbones/__init__.py

```diff
@@ -17,5 +17,6 @@
 from .efficientnet import OTXEfficientNet
 from .efficientnetv2 import OTXEfficientNetV2
 from .mobilenetv3 import OTXMobileNetV3
+from .torchvision_backbones import *
```

Comment:
If you add this code for executing torchvision_backbones.py, then you can just add code like bellow

`from . import torchvision_backbones`

Reply:
thanks for the nice comment! Edited.